packages/flutter/test/material/checkbox_list_tile_test.dart

```diff
@@ -1119,14 +1119,16 @@ void main() {
       await tester.pumpWidget(buildApp(platform));
       await tester.pumpAndSettle();
 
-      expect(find.byType(CupertinoCheckbox), findsOneWidget);
+      expect(find.byType(CupertinoCheckbox), findsNothing);
+      expect(find.byType(Checkbox), findsOneWidget);
     }
 
     for (final TargetPlatform platform in <TargetPlatform>[ TargetPlatform.android, TargetPlatform.fuchsia, TargetPlatform.linux, TargetPlatform.windows ]) {
       await tester.pumpWidget(buildApp(platform));
       await tester.pumpAndSettle();
 
       expect(find.byType(CupertinoCheckbox), findsNothing);
+      expect(find.byType(Checkbox), findsOneWidget);
```

Comment:
Can you combine these two `for` loops now?

Reply:
If we are no longer delegating to `CupertinoCheckbox`, the test seems unnecessary (unless we change it to test that we no longer delegate to `CupertinoCheckbox`